content/en/blog/_posts/2024-08-15-kubernetes-1.31-image-volume-source.md

```diff
@@ -0,0 +1,186 @@
+the model-server container image. They can package these in an OCI object to
+take advantage of OCI distribution and ensure efficient model deployment. This
+allows them to separate the model specifications/content from the executables
+that process them.
+
+Another use case is that security engineers can use a public image for a malware
+scanner and mount in a volume of private (commercial) malware signatures, so
+that they can load those signatures without baking their own combined image (which
+might not be allowed by the copyright on the public image). Those files work
+regardless of the OS or version of the scanner software.
+
+But in the long term it will be up to **you** as an end user of this project to
+outline further important use cases for the new `ImageVolume` feature.
+[SIG Node](https://github.com/kubernetes/community/blob/54a67f5/sig-node/README.md)
+is happy to retrieve any feedback or suggestions for further enhancements to
+allow more advanced usage scenarios. Feel free to provide feedback by either
+using the [Kubernetes Slack (#sig-node)](https://kubernetes.slack.com/messages/sig-node)
+channel or the [SIG Node mailinglist](https://groups.google.com/g/kubernetes-sig-node).
+
+## Detailed Example
+
+The Kubernetes alpha feature gate [`ImageVolume`](/docs/reference/command-line-tools-reference/feature-gates)
+needs to be enabled on the [API Server](/docs/reference/command-line-tools-reference/kube-apiserver)
+as well as the [kubelet](/docs/reference/command-line-tools-reference/kubelet)
+to make it functional. If that’s the case and the [container runtime](/docs/setup/production-environment/container-runtimes)
+has support for the feature (like CRI-O ≥ v1.31), then an example `pod.yaml`
+like this can be created:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod
+spec:
+  containers:
+    - name: test
+      image: registry.k8s.io/e2e-test-images/echoserver:2.3
+      volumeMounts:
+        - name: volume
+          mountPath: /volume
+  volumes:
+    - name: volume
+      image:
+        reference: quay.io/crio/artifact:v1
+        pullPolicy: IfNotPresent
+```
+
+The pod declares a new volume using the `image.reference` of
+`quay.io/crio/artifact:v1`, which refers to an OCI object containing two files.
+The `pullPolicy` behaves in the same way as for container images and allows the
+following values:
+
+- `Always`: the kubelet always attempts to pull the reference and the container
+  creation will fail if the pull fails.
+- `Never`: the kubelet never pulls the reference and only uses a local image or
+  artifact. The container creation will fail if the reference isn’t present.
+- `IfNotPresent`: the kubelet pulls if the reference isn’t already present on
+  disk. The container creation will fail if the reference isn’t present and the
+  pull fails.
+
+The `volumeMounts` field is indicating that the container with the name `test`
+should mount the volume under the path `/volume`.
+
+If we now create the pod:
```

Comment:
(nit) Better to [address the reader as “you”](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you)